package.json

```diff
@@ -7,52 +7,40 @@
   "license": "Apache-2.0",
   "type": "module",
   "scripts": {
-    "all": "concurrently -g pnpm:check-* pnpm:test pnpm:build",
+    "all": "concurrently -g pnpm:lint pnpm:test pnpm:build",
```

Comment:
intentional to no longer run `pnpm check` and `pnpm format` here?

Reply:
Thanks for the callout - I had not updated the README properly. But yes, this was an intentional alignment with app

- `pnpm:lint` now runs both ESLint and Prettier
- `pnpm:build` now runs the type checks as part of building the TS output

So end result: `all` still checks ESLint, Prettier, TypeScript, and tests